rct229/reports/ashrae901_2019_software_test_report.py

```diff
@@ -0,0 +1,76 @@
+import json
+from copy import deepcopy
+from datetime import datetime
+
+from rct229.report_engine.rct_report import RCTReport
+from rct229.reports.utils import calc_vals_converter
+from rct229.rule_engine.rct_outcome_label import RCTOutcomeLabel
+
+
+class ASHRAE9012019SoftwareTestReport(RCTReport):
+    def __init__(self):
+        super(ASHRAE9012019SoftwareTestReport, self).__init__()
+        self.title = "ASHRAE STD 229P RULESET CHECKING TOOL"
+        self.purpose = "RCT Ruleset Software Testing Report"
+        self.ruleset = "ASHRAE 90.1-2019 Performance Rating Method (Appendix G)"
+        self.schema_version = "0.0.23"
+        self.date_run = str(datetime.now())
+        self.ruleset_report_file = "ashrae901_2019_software_testing_report.json"
+
+    def initialize_ruleset_report(self):
+        report_json = dict()
+        report_json["title"] = self.title
```

Comment:
This is a great way to set the values. Another way could be (and can delete line 21 - just as a reference)
```python
report_json = {
"title":  self.title,
"purpose": self.purpose,
 "ruleset": self.ruleset,
"date_run": self.date_run,
"schema_version": self.schema_version,
"rule_tests": [],}
```
Also, depending on your coding style, I'll leave this up to you!